src/index.html

```diff
@@ -10,7 +10,33 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page">
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form
+      action="#"
+      data-qa="big"
+      class="search-bar page__search search-bar--big"
+    >
+      <input
+        type="text"
+        class="search-bar__text-input search-bar__text-input--big"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles”"
+      >
+    </form>
+    <form
+      action="#"
+      class="
+        search-bar
+        page__search
+        search-bar--small"
```

Comment:
It's a class, you can put it in 1 line
```suggestion
      class="search-bar page__search search-bar--small"
```